Doc/library/concurrent.futures.rst

```diff
@@ -151,8 +153,10 @@ And::
        print(f.result())
 
    executor = ThreadPoolExecutor(max_workers=1)
-   executor.submit(wait_on_future)
-
+   executor.submit(wait_on_future).result()
```

Comment:
We should not call `.result()`. We should indicate in the comment that calling `.result` would produce a deadlock.